src/mixins/canvas_events.mixin.js

```diff
@@ -50,11 +50,12 @@
     },
 
     addOrRemove: function(functor, eventjsFunctor) {
+      var eventTypePrefix = this.get('enablePointerEvents') ? 'pointer' : 'mouse';
```

Comment:
please use `this.enablePointerEvents` for consistency with fabric codebase in general

Reply:
ok! done in 96a483543e3e5e163a50a1771a2d6ea9d6d1a02a :)